avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -446,3 +446,16 @@ unitpfv = ms-1
 # peak files and plots are exported, option to turn off exports when exportData is set to False
 # this affects export of peak files and also generation of peak file plots
 exportData = True
+
+
+[distanceTimeAnalysis_override]
```

Comment:
Mh, I'm not sure this is a good idea. I think this module should be purely computation and only have a flag that produces data for another module, but not have override for that...

Reply:
this is changed now - one function to create new local config files for override modules